WordPress/Classes/Services/WordPressXMLRPCApiFacade.h

```diff
@@ -0,0 +1,19 @@
+#import <Foundation/Foundation.h>
+
+@protocol WordPressXMLRPCApiFacade
```

Comment:
Same as above: I think we can avoid this class since the method it contains can be part of our API classes.